donate/templates/pages/core/contributor_support_page_master.html

```diff
@@ -7,18 +7,24 @@
 {% endblock %}
 
 {% block head_addenda %}
-    <script src="https://www.google.com/recaptcha/api.js"></script>
-    <script>
-        function timestamp() {
-            var response = document.getElementById("g-recaptcha-response");
-            if (response == null || response.value.trim() == "") {
-                var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);
-                elems["ts"] = JSON.stringify(new Date().getTime());
-                document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems);
-            }
+    <script type="text/javascript">
+    function hookupFormSubmitButton() {
+        try {
+            const recaptcha = document.getElementById("g-recaptcha");
+            console.log(recaptcha);
+            grecaptcha.render(recaptcha, {
+                sitekey: "{{ help_recaptcha_site_key }}",
+                callback: (token) => {
+                    var submitButton = document.querySelector('.support-form input[type="submit"]');
+                    submitButton.removeAttribute("disabled");
+                }
+            });
+        } catch (e) {
+            console.error(e);
         }
-        setInterval(timestamp, 500);
+      };
     </script>
+    <script src="https://www.google.com/recaptcha/api.js?onload=hookupFormSubmitButton&render=explicit" async defer></script>
```

Comment:
this switches the recaptcha over to running the `hookupFormSubmitButton` function after it boostraps itself